tests/users/test_resend_email.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import uuid
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework.test import APITestCase
+from unittest.mock import patch
+from users.models import User, UserRole
+
+logger = logging.getLogger(__name__)
```

Comment:
The logger is defined here, but this is a test file. Excessive logging in tests can produce noisy output. Consider using logging only for debugging or critical failures, and possibly remove info-level logs.